microsite/data/plugins/rollbar.yaml

```diff
@@ -0,0 +1,8 @@
+---
+title: Rollbar
+author: roadie.io
```

Comment:
Category isn't part of the documentation above?

Reply:
Just to point out, it's totally my fault that this got misattributed in this PR. Iain only started working with Backstage this week and hasn't seen the history of how developed each plugin. I suggested we use these three plugins as sample data when working on this feature and I failed to properly explain who had developed which one. It's totally my bad and it will be fixed ASAP.

Reply:
Sorry @andrewthauer.

Regarding the category I'll added it to the UI and to the docs.

Reply:
No problem @iain-b